planner/core/testdata/integration_suite_out.json

```diff
@@ -2485,172 +2485,171 @@
-          "  └─Projection_38 3.00 76.80 mpp[tiflash]  Column#4",
-          "    └─HashAgg_36 3.00 57.00 mpp[tiflash]  group by:test.t2.a, funcs:count(1)->Column#4",
-          "      └─ExchangeReceiver_22 3.00 48.00 mpp[tiflash]  ",
-          "        └─ExchangeSender_21 3.00 45.00 mpp[tiflash]  ExchangeType: HashPartition, Hash Cols: [name: test.t2.a, collate: binary]",
-          "          └─TableFullScan_20 3.00 45.00 mpp[tiflash] table:t2 keep order:false"
+          "TableReader_25 3.00 25896.21 root  data:ExchangeSender_24",
+          "└─ExchangeSender_24 3.00 388390.38 mpp[tiflash]  ExchangeType: PassThrough",
+          "  └─Projection_23 3.00 388390.38 mpp[tiflash]  Column#4",
+          "    └─HashAgg_9 3.00 388390.38 mpp[tiflash]  group by:test.t2.a, funcs:count(1)->Column#4",
+          "      └─ExchangeReceiver_22 3.00 385507.26 mpp[tiflash]  ",
+          "        └─ExchangeSender_21 3.00 385459.26 mpp[tiflash]  ExchangeType: HashPartition, Hash Cols: [name: test.t2.a, collate: binary]",
+          "          └─TableFullScan_20 3.00 385459.26 mpp[tiflash] table:t2 keep order:false"
         ]
       },
       {
         "SQL": "explain format = 'verbose' select count(*) from t3 where b = 0",
         "Plan": [
-          "StreamAgg_10 1.00 1.33 root  funcs:count(1)->Column#4",
-          "└─IndexReader_15 0.00 1.33 root  index:IndexRangeScan_14",
+          "StreamAgg_10 1.00 0.00 root  funcs:count(1)->Column#4",
+          "└─IndexReader_15 0.00 0.00 root  index:IndexRangeScan_14",
           "  └─IndexRangeScan_14 0.00 0.00 cop[tikv] table:t3, index:c(b) range:[0,0], keep order:false"
         ]
       },
       {
         "SQL": "explain format = 'verbose' select /*+ use_index(t3, c) */ count(a) from t3 where b = 0",
         "Plan": [
-          "StreamAgg_10 1.00 19.33 root  funcs:count(test.t3.a)->Column#4",
-          "└─IndexLookUp_17 0.00 19.33 root  ",
+          "StreamAgg_10 1.00 0.00 root  funcs:count(test.t3.a)->Column#4",
+          "└─IndexLookUp_17 0.00 0.00 root  ",
           "  ├─IndexRangeScan_15(Build) 0.00 0.00 cop[tikv] table:t3, index:c(b) range:[0,0], keep order:false",
           "  └─TableRowIDScan_16(Probe) 0.00 0.00 cop[tikv] table:t3 keep order:false"
         ]
       },
       {
         "SQL": "explain format = 'verbose' select count(*) from t2 where a = 0",
         "Plan": [
-          "StreamAgg_12 1.00 4.93 root  funcs:count(1)->Column#4",
-          "└─TableReader_24 0.00 4.93 root  data:Selection_23",
-          "  └─Selection_23 0.00 54.00 cop[tiflash]  eq(test.t2.a, 0)",
-          "    └─TableFullScan_22 3.00 45.00 cop[tiflash] table:t2 keep order:false"
+          "StreamAgg_12 1.00 47.30 root  funcs:count(1)->Column#4",
+          "└─TableReader_21 0.00 47.30 root  data:Selection_20",
+          "  └─Selection_20 0.00 709.52 cop[tikv]  eq(test.t2.a, 0)",
+          "    └─TableFullScan_19 3.00 559.82 cop[tikv] table:t2 keep order:false"
```

Comment:
Expected since model2 prefers to use TiKV-Scan instead of TiFlash-Scan when est-row is small.